packages/esm-service-queues-app/src/transition-latest-queue-entry/add-patient-to-queue-modal/add-patient-to-queue-entry.scss

```diff
@@ -0,0 +1,46 @@
+@use '@carbon/layout';
+@use '@carbon/type';
+@use '@openmrs/esm-styleguide/src/vars' as *;
+
+.modalHeader {
+  :global {
+    .cds--modal-close-button {
+      position: relative;
+      inset-block-start: 0;
+      inset-inline-end: 0;
+      margin: 0;
+      margin-top: calc(-1 * #{layout.$spacing-05});
+    }
+
+    .cds--modal-close {
+      background-color: rgba(0, 0, 0, 0);
+
+      &:hover {
+        background-color: var(--cds-layer-hover);
+      }
+    }
+
+    .cds--popover--left > .cds--popover > .cds--popover-content {
+      transform: translate(-4rem, 0.85rem);
+    }
+
+    .cds--popover--left > .cds--popover > .cds--popover-caret {
+      transform: translate(-3.75rem, 1.25rem);
+    }
+  }
+}
```

Comment:
```suggestion
```
This is no longer necessary since we upgraded Carbon. With this in place, notice how the close button is not where it should be:
<img width="820" alt="Screenshot 2025-05-16 at 12 34 26 AM" src="https://github.com/user-attachments/assets/837f1948-6e2d-431a-8f93-9e0cbd7da242" />